docs/troubleshooting.md

```diff
@@ -19,9 +19,21 @@ provide you with better support on your issue.
 This error might be caused by a memory allocation failure, and it fails
 silently. You can fix this error by decreasing your batch size and trying again.
 
-`plaidml.exceptions.ResourceExhausted: Out of memory`
+`plaidml.exceptions.ResourceExhausted: Out of memory` 
+
 This error is caused by incorrect Tile syntax.
 
+### Bazel Issues
+
+`Encountered error while reading extension file 'workspace.bzl': no such package
+'@toolchain//'`
+
+On MacOS devices, `toolchain` errors often indicate that the user does not have
+Xcode properly installed. Even if you have Xcode Command Line Tools installed,
+you may not have a proper installation of Xcode itself. You can verify your
+Xcode installation by running `spctl --assess --verbose
+/Applications/Xcode.app`.
```

Comment:
I think we should point people to the bazel docs in this case:

https://docs.bazel.build/versions/0.28.0/install-os-x.html

Also, I think what we can do is say, run: 
```
xcode-select -p
``` 

And if it doesn't say `/Applications/Xcode.app/Contents/Developer` then go to the App Store and install Xcode. Then be sure to use:
```
bazelisk clean --expunge
```
before trying to use bazel again.

Reply:
`spctl` just hangs for me, but why are we trying to asses something that might not yet exist? For users without Xcode installed, they won't have a `/Application/Xcode.app` directory.

Reply:
Fair point - I'm modifying the docs to address this.